projects/WeTek_Hub/linux/linux.aarch64.conf

```diff
@@ -34,7 +39,7 @@ CONFIG_CROSS_COMPILE=""
 CONFIG_LOCALVERSION=""
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_DEFAULT_HOSTNAME="@DISTRONAME@"
-CONFIG_SWAP=y
+# CONFIG_SWAP is not set
```

Comment:
I think it is better to not introduce implicit changes. If you want do disable swap you should do separate commit and explain why toy did that.